app/api/video/tests/test_api/test_api_video.py

```diff
@@ -0,0 +1,109 @@
+import json
```

Comment:
why not use pytest

Reply:
in my opinion django and django-rest-framework provide good unit test, but if you think that pytest looks better i will update to pytest